src/commands/script/execute/index.ts

```diff
@@ -125,27 +129,31 @@ export const executeActionCallbackHandler = (spinner: Ora, filename: string, pro
  * Handlers
  */
 
-export const handleExecuteScript = async (_fileName?: string) => {
+export const handleExecuteScript = async (_fileName?: string, option: HandleExecuteScriptOption = {}) => {
   ensureCwdRootProject()
 
   const scriptFileDirectory = getScriptFileDirectory()
-  let fileName = _fileName
+  let fileName: string | undefined = _fileName
 
   if (!fileName) {
     const currentFileList = getFilesCurrentDir(scriptFileDirectory)
-    if (currentFileList.length === 0) {
+    const currentTsFileList = currentFileList.filter((v) => v.endsWith('.ts'))
```

Comment:
`.ts` file will also be included in the list in this case which it isn't the ts file